src/EditorFeatures/VisualBasicTest/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.vb

```diff
@@ -89,6 +89,38 @@ End Class",
 index:=1)
         End Function
 
+        <WorkItem(30396, "https://github.com/dotnet/roslyn/issues/30396")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)>
+        Public Async Function TestStructure() As Task
```

Comment:
💡 An additional test for already implementing `IEquatable(Of Z)` would be good

Reply:
done.